docs/source/api/diffsync.rst

```diff
@@ -0,0 +1,56 @@
+API Reference
+=============
+
+.. automodule:: diffsync
+   :members:
+   :undoc-members:
+   :show-inheritance:
+
+
+diffsync.diff
```

Comment:
I'd consider rendering each module to a separate docs page rather than putting the entire API in a single page - I just think it makes for a very long page.

Reply:
I was able to break these out into separate pages in the latest commit.